internal/models/device.go

```diff
@@ -54,4 +54,5 @@ type UpdateDevice struct {
 	Hostname       string     `json:"hostname" example:"myhost"`
 	Endpoints      []Endpoint `json:"endpoints" gorm:"type:JSONB; serializer:json"`
 	Revision       *uint64    `json:"revision"`
+	Relay          bool       `json:"relay"`
```

Comment:
Can we make this a *bool so that updating the relay status is optional?

Reply:
Done